README.md

```diff
@@ -3,63 +3,70 @@
 [![test](https://github.com/blocoio/android-template/workflows/test/badge.svg?branch=master)](https://github.com/blocoio/android-template/actions?query=workflow%3Atest+branch%3Amaster)
 [![lint](https://github.com/blocoio/android-template/workflows/lint/badge.svg?branch=master)](https://github.com/blocoio/android-template/actions?query=workflow%3Alint+branch%3Amaster)
 
-## Clean architecture with 3 layers
+The Goal of this project is to have a starting point when starting a new project that follows the best development practices, this project is our interpretation and in-house solution to meet the official [architecture](https://developer.android.com/topic/architecture) guidelines provided by Google.
```

Comment:
A suggestion for a simpler version of this introduction:

The goal of this Template is to be our starting point for new projects, following the best development practices. It's our interpretation and adaptation of the official [architecture](https://developer.android.com/topic/architecture) guidelines provided by Google. And it's inspired by Google's [NowInAndroid](https://github.com/android/nowinandroid).